woocommerce/woocommerce
# [Accessibility] Add scope attribute to the product attributes table

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43608 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Visit a Product Detail page, and select the Additional Information Tab
2. Inspect the table to make sure that the row headings have a scope of "row"
3. In addition, a label was added to the table, inspect to make sure that the `aria-label` is "Product Details"
4. Using a screenreader, navigate to the table (Mac VoiceOVer .. use the Rotor to easily and quickly navigate to the table)
5. Navigate through the table and make sure that the first time landing in a table detail, the header is also read off. 
_If the table only has one row, the table detail may not be read as the user will have just heard that single row's heading announced as they navigate through the table._



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
